packages/vite-plugin-web-extension/src/options.ts

```diff
@@ -62,7 +62,7 @@ export interface UserOptions {
    *
    * @default "chrome"
    */
-  browser?: string;
+  browser?: 'chrome' | 'firefox' | (string & {}) | null;
```

Comment:
this is a TypeScript trick that provides auto-completion for some values while still allowing any string

Reply:
Interesting, didn't know this worked... I've only used branded types with a property inside the `{}`. I've always had to use a function overload to get this behavior before, very good to know!